features/user_create_new_recipe.feature

```diff
@@ -0,0 +1,34 @@
+Feature: User creates new recipe
+  As a user
+  In order to contribute to the site
+  I should be able to add new recipe
+
+  Background:
+    Given We have the following user:
+      | email          | password |
+      | maran@test.com | 12345678 |
+
+    And I visit the login page
+    And I fill in "Email" with "maran@test.com"
+    And I fill in "Password" with "12345678"
+    And I click "Log in"
+    And I visit the site
```

Comment:
Please ide Warren test helpers to login in scenarios where you need a current_user instance

Reply:
Ok will do

Reply:
Done